src/libraries/System.IO.FileSystem/tests/File/Append.cs

```diff
@@ -77,4 +77,23 @@ public void NullEncoding()
             Assert.Throws<ArgumentNullException>(() => File.AppendAllLines(GetTestFilePath(), new string[] { "Text" }, null));
         }
     }
+
+    public class File_AppendAllBytes: File_ReadWriteAllBytes
+    {
+        [Fact]
+        public void ValidParameters_ShouldAppendBytesToFile()
+        {
+            string path = GetTestFilePath();
+            byte[] originalBytes = new byte[] { 1, 2, 3 };
+            byte[] appendBytes = new byte[] { 4, 5, 6 };
+
+            File.WriteAllBytes(path, originalBytes);
+            File.AppendAllBytes(path, appendBytes);
+
+            byte[] expectedBytes = new byte[] { 1, 2, 3, 4, 5, 6 };
+            byte[] actualBytes = File.ReadAllBytes(path);
+            Assert.Equal(expectedBytes, actualBytes);
+            File.Delete(path);
+        }
+    }
```

Comment:
You need to include more input validation. What happens if you receive `null`? What about a path to a non-existing file?